recipes/mne-realtime/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "mne-realtime" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7eb763dc75ad45572b7d3a7e26b08503ece6bcc9bdea82d8dde95b8ca339ad23
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
```

Comment:
I cannot find anything about 3.7 upstream. Where is that mentioned?

Reply:
That's a bug -- @hoechenberger I fixed it and pushed 0.1.2 to PyPi, can you update?

Reply:
I don't think it's explicitly mentioned anywhere, but I have to put a constraint here since it's a `noarch` package. Pratically, MNE-Realtime only works with Python 3.7+, as it depends on MNE-Python, which in turn requires Python 3.7 and up

Reply:
@larsoner I've bumped it to 0.1.2

Reply:
If a dependency has a python requirement, it doesn't need to be added here. Only the minimum python of this package should be listed. The solver will make sure that the minimum python version is pulled in that can be supported by all dependencies and the package itself.
The reason for this is that incase MNE-Python ever has a lower bound in the future, you don't need to change it here, changing it there is sufficient.

But I saw that upstream is now has it as well so let's just keep it. :)

Reply:
Thanks!